### ISSUE TITLE
Allow specific hour/day range combinations to be tracked against a specific tag/label

### ISSUE THREAD

**Issue by geekpete** [PaulMorris/mind-the-time#7]
It would be nice to only track usage between specific time frames and/or days.
Rather than not counting, it'd be even more useful to see usage within one time range vs outside that timerange.

So thinking work hours vs after work potentially.
Would just need a setting to specify a label and the hours/days ranges that apply to that label.
This would allow whatever ranges to be tracked just by tagging them with a label then aggregating the labels individually.
Would allow overlaps then too.

**Comment by geekpete** [comment]
How about logging to disk as a starting point? I could then index that data into something like Elasticsearch and go bananas with it. The tricky UI parts for the plugin can come later potentially, but with a way to externalise the data, the plugin will be immediately useful for any number of uses.

**Comment by PaulMorris** [comment]
That could be a good starting point.  I'm not sure what form of 'logging to disk' you mean in this case though, can you say more?

**Comment by geekpete** [comment]
Appending to a CSV would be something another tool could ingest

**Comment by PaulMorris** [comment]
Okay, makes sense, assuming there is a way to do that from a web extension add-on.

**Comment by geekpete** [comment]
Good point, there's going to probably be numerous security barriers involved.
Is it any easier if you can log to an external API?
